README.md

```diff
@@ -26,6 +26,8 @@ key.converter|org.apache.kafka.connect.json.JsonConverter
 value.converter|org.apache.kafka.connect.json.JsonConverter
 key.converter.schemas.enable|true
 value.converter.schemas.enable|true
+offset.storage.file.filename|storage
+bootstrap.servers|localhost:9092
```

Comment:
I think we need to make a distinction here between what is required for Kafka vs what is reqiured for our connector.

How about something like:
```markdown
The Cloudant Kafka connector can be configured in standalone or distributed mode according to the [Kafka Connector documentation](http://docs.confluent.io/3.0.1/connect/userguide.html#configuring-connectors). At a minimum It is necessary to configure:

1. `bootstrap.servers`
2. If using a standalone worker `offset.storage.file.filename`.
3. The following configuration when using the Cloudant connector as either a source or a sink:

Parameter | Value
---:|:---
key.converter|org.apache.kafka.connect.json.JsonConverter
value.converter|org.apache.kafka.connect.json.JsonConverter
key.converter.schemas.enable|true
value.converter.schemas.enable|true
```

Reply:
Fixed in 5dde480